addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -161,18 +161,38 @@
                             <t t-call="l10n_it_edi.account_invoice_line_it_FatturaPA"/>
                         </t>
-                        <t t-foreach="record.line_ids.filtered(lambda line: line.tax_line_id)" t-as="tax_line">
-                            <DatiRiepilogo>
-                                <!--2.2.2-->
-                                <AliquotaIVA t-esc="format_numbers(tax_line.tax_line_id.amount)"/>
-                                <Natura t-if="tax_line.tax_line_id.l10n_it_has_exoneration" t-esc="tax_line.tax_line_id.l10n_it_kind_exoneration"/>
-                                <ImponibileImporto t-esc="format_monetary(tax_line.tax_base_amount, currency)"/>
-                                <Imposta t-esc="format_monetary(tax_line.price_unit, currency)"/>
-                                <EsigibilitaIVA t-if="not tax_line.tax_line_id.l10n_it_has_exoneration or tax_line.tax_line_id.l10n_it_kind_exoneration=='N6'" t-esc="tax_line.tax_line_id.l10n_it_vat_due_date"/>
-                                <RiferimentoNormativo t-if="tax_line.tax_line_id.l10n_it_has_exoneration" t-esc="tax_line.tax_line_id.l10n_it_law_reference"/>
-                            </DatiRiepilogo>
+                        <t t-if="record.line_ids.filtered(lambda line: line.tax_line_id)">
+                            <t t-foreach="record.line_ids.filtered(lambda line: line.tax_line_id)" t-as="tax_line">
+                                <DatiRiepilogo>
+                                    <!--2.2.2-->
+                                    <AliquotaIVA t-esc="format_numbers(tax_line.tax_line_id.amount)"/>
+                                    <Natura t-if="tax_line.tax_line_id.l10n_it_has_exoneration" t-esc="tax_line.tax_line_id.l10n_it_kind_exoneration"/>
+                                    <ImponibileImporto t-esc="format_monetary(tax_line.tax_base_amount, currency)"/>
+                                    <Imposta t-esc="format_monetary(tax_line.price_unit, currency)"/>
+                                    <EsigibilitaIVA t-if="not tax_line.tax_line_id.l10n_it_has_exoneration or tax_line.tax_line_id.l10n_it_kind_exoneration=='N6'" t-esc="tax_line.tax_line_id.l10n_it_vat_due_date"/>
+                                    <RiferimentoNormativo t-if="tax_line.tax_line_id.l10n_it_has_exoneration" t-esc="tax_line.tax_line_id.l10n_it_law_reference"/>
+                                </DatiRiepilogo>
+                            </t>
+                        </t>
+                        <t t-else="">
+                            <!-- invoice with 0% tax only -->
+                            <t t-foreach="record.line_ids.mapped('tax_ids')" t-as="tax">
+                                <t t-set="amount" t-value="0">
+                                <t t-foreach="record.line_ids.filtered(lambda l: l.tax_ids == tax)" t-as="line">
+                                    <t t-set="taxes" t-value="line.tax_ids.compute_all(line.price_unit)"/>
```

Comment:
wrong as you need the quantity here and not sure what you want to accomplish with the for each here.